app/controllers/payments_controller.rb

```diff
@@ -10,6 +10,9 @@ def index
 
   def new
     @total_amount_to_pay = Ticket.total_price(@conference, current_user, paid: false)
+    if @conference.end_date < Date.today
```

Comment:
I think this needs to be added in ability.rb and let cancancan handle things from there. 
This is not a link we would use directly anywhere, anyway, so we care less about the message. Cancancan will show the default message 'You are not authorized to access this page'